SimakovaKY/SimakovaKY_lab_2

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+using namespace std;
+int main() {
+    const int size = 7;
+    int arr[size];
+    float sr_znach;
+    int max, min;
+    int per = 0;
+    int i, j;
+   
+      max = -1000000;
+      min = 1000000;
+      sr_znach = 0;
+    for (int i = 0; i < size; i++) {
+
+        arr[i] = rand();
+       cout << arr[i]<< ' ';  
+    }
+
+ cout << endl;
+    
+      for (j = 0; j < size; j++) {
+        per = arr[j];
+        i = j - 1;
+         while (i >= 0 && arr[i] > per ) {
+            arr[i + 1] = arr[i];
+            i = i - 1;
+            arr[i + 1] = per;
+         }
+      }
+
+    for (i = 0; i < size; i++) {
+        cout << arr[i]<<' ';
+    }
+
+ cout << endl;
+
+      for (i = 0; i < size; i++) {
+            if (arr[i] % 2 != 0) {
+            cout << arr[i] << ' ';
+            }
+       }
+
+ cout << endl;
+
+      for (i = 0; i < 7; i++) {
```

Comment:
You have size constant that contains size of array

Reply:
Can you explain how to create a folder? I don't understand what i'm doing wrong.